projects/website/markdown/docs/commands/up.mdx

```diff
@@ -111,6 +111,37 @@ Besides the built-in commands, you can define yours using the custom option. It
 
 - **Name:** Name of the command
 - **Meet:** Command to run in the system to configure the environment.
-- **Met:** Command to run in the system to verify whether the environment is already configure. A 0 exit code means that the environment is already configured.
+- **Met:** Command to run in the system to verify whether the environment is already configured. A 0 exit code means that the environment is already configured.
+
+### Precondition
+
+```swift
+  .precondition(
+      name: "Check GITHUB_TOKEN",
+      advice: "“GITHUB_TOKEN” environment variable must be set.",
+      isMet: ["scripts/validations/github_token.py"]
+  )
+```
+
+```python
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+
+try:
+    t = os.environ["GITHUB_TOKEN"]
+except:
+    sys.exit(1)
+
+sys.exit(0)
+```
```

Comment:
Thoughts on keeping this out and just leaving the paragraph below?

Reply:
Because I spent so much time failing to make the `UpCustom` work, I wanted to give a concrete example of how this did work. I'm not against removing it.